src/PartitionService.ts

```diff
@@ -53,6 +54,12 @@ export interface PartitionService {
 
 }
 
+interface ClientForPartitionService {
```

Comment:
This seems to be redundant as `PartitionService` doesn't seem to have cyclic depedencies with other modules.

Reply:
You're right thanks